src/test/java/com/zufar/onlinestore/security/api/UserRegistrationServiceTest.java

```diff
@@ -0,0 +1,73 @@
+package com.zufar.onlinestore.security.api;
+
+import com.zufar.onlinestore.openapi.dto.UserDto;
+import com.zufar.onlinestore.security.converter.RegistrationDtoConverter;
+import com.zufar.onlinestore.security.dto.UserRegistrationRequest;
+import com.zufar.onlinestore.security.jwt.JwtTokenProvider;
+import com.zufar.onlinestore.user.api.UserApi;
+import com.zufar.onlinestore.user.converter.UserDtoConverter;
+import com.zufar.onlinestore.user.converter.UserDtoConverterImpl;
+import com.zufar.onlinestore.user.entity.UserEntity;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.any;
+
+@ExtendWith(MockitoExtension.class)
+class UserRegistrationServiceTest {
+    @InjectMocks
+    private UserRegistrationService userRegistrationService;
+    @Mock
+    private UserApi userApi;
+    @Mock
+    private RegistrationDtoConverter registrationDtoConverter;
+    @Mock
+    private UserDtoConverter userDtoConverter;
+    @Mock
+    private JwtTokenProvider jwtTokenProvider;
+    @Mock
+    private PasswordEncoder passwordEncoder;
+
+    @Test
+    @DisplayName("mock test register")
+    void testRegister() {
+        UserRegistrationRequest request = new UserRegistrationRequest("name","lastName", "email", "password");
+        UserDto userDto = new UserDto();
```

Comment:
btw u can take a look at a library weve been using for preparing some dumb stubs. It calls -  Inctancio